src/service/position-observer/position-observer-worker.js

```diff
@@ -40,7 +40,7 @@ const LOW_FIDELITY_FRAME_COUNT = 4;
  * @typedef {{
  *  positionRect: ?../../layout-rect.LayoutRectDef,
  *  viewportRect: !../../layout-rect.LayoutRectDef,
- *  relativePos: string,
+ *  relativePos: ?../../layout-rect.RelativePositions,
```

Comment:
Can we keep the `RelativePositions` enum?

Reply:
Hmm not sure I understand, this doesn't remove it

Reply:
lol didn't know how but I was confused by the diff. Ignore my comment : ) LGTM Thanks!